package.json

```diff
@@ -12,8 +12,32 @@
   "author": "OpenLaw Team <opensource@openlaw.io>",
   "license": "CC-BY-4.0",
   "devDependencies": {
+    "eslint": "^5.14.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-standard": "^4.0.0",
     "prettier": "^1.16.4",
     "prettier-check": "^2.0.0",
-    "vuepress": "^0.14.8"
-  }
+    "vuepress": "^0.14.9"
+  },
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "dependencies": {},
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mapachurro/docs.git"
+  },
+  "keywords": [
+    "i18n",
+    "markup",
+    "spanish"
+  ],
+  "bugs": {
+    "url": "https://github.com/mapachurro/docs/issues"
+  },
+  "homepage": "https://github.com/mapachurro/docs#readme"
 }
```

Comment:
Looks like this was carried over from your forked repo. I think everything below the `vuepress` update can be omitted before merging?

Reply:
Copy that. Just pushed up a version of the file without that stuff.

Reply:
Ok thanks for the update there.